dissect/target/plugins/apps/vpns/wireguard.py

```diff
@@ -63,21 +64,21 @@ class WireGuardPlugin(Plugin):
           - C:\\Program Files\\WireGuard\\Data\\Configurations
     """
 
-    os_config_globs = {
-        OperatingSystem.UNIX.value: ["/etc/wireguard/*.conf"],
-        OperatingSystem.LINUX.value: ["/etc/wireguard/*.conf"],
-        OperatingSystem.OSX.value: ["/usr/local/etc/wireguard/*.conf", "/opt/homebrew/etc/wireguard/*.conf"],
-        OperatingSystem.WINDOWS.value: [
-            r"C:\Windows\System32\config\systemprofile\AppData\Local\WireGuard\Configurations\*.dpapi",
-            r"C:\Program Files\WireGuard\Data\Configurations\*.dpapi",
-        ],
-    }
+    CONFIG_GLOBS = [
+        # Linux
+        "/etc/wireguard/*.conf",
+        # MacOS
+        "/usr/local/etc/wireguard/*.conf",
+        "/opt/homebrew/etc/wireguard/*.conf",
+        # Windows
+        "C:\\Windows\\System32\\config\\systemprofile\\AppData\\Local\\WireGuard\\Configurations\\*.dpapi",
+        "C:\\Program Files\\WireGuard\\Data\\Configurations\\*.dpapi",
+    ]
 
     def __init__(self, target) -> None:
         super().__init__(target)
-        self.configs = []
-        config_globs = self.os_config_globs.get(target.os, [])
-        for path in config_globs:
+        self.configs: list[Path] = []
+        for path in self.CONFIG_GLOBS:
```

Comment:
Is there a specific reason to attempt to read all files, instead of just the relevant ones?

Reply:
Mostly readability and ease of expansion. Especially the Unix-y based paths would quickly get hairy and easy to forget when adding new operating system support to Dissect (e.g. BSD). It's a small amount of paths so I feel it's okay to "bruteforce" it.